index.html

```diff
@@ -9,25 +9,62 @@
     <title>AwesomeBooks</title>
   </head>
 <body>
-  <h1 class="d-flex justify-content-center">All Awesome books</h1>
-    <div id="bookshelf"></div>
-    <br />
-    <form id="addbook" class="d-flex justify-content-center">
-      <h2>Add a new book</h2>
-      <div>
-        <label for="title"></label>
-        <input type="text" id="title" placeholder="Title" />
-      </div>
-      <br />
-      <div>
-        <label for="author"></label>
-        <input type="text" id="author" placeholder="Author" />
-      </div>
-      <br />
-      <button id="btn" >Add</button>
-    </form>
-    <script src="./scripts.js"></script>
-  </body>
+<header>
+  <nav class="navigation d-flex justify-content-between align-items-center">
+    <h2 class="logo ms-3">Awesome Books</h2>
+    <ul class="d-flex gap-3">
+      <li><a id="booksBtn" href="#">List</a></li>
+      <li><a id="addbookBtn" href="#">Add NewBook</a></li>
+      <li><a id="contactBtn" href="#">Contact</a></li>
+    </ul>
+  </nav>
+</header>
+
+<main>
+  <div class="clock d-flex gap-3 justify-content-end me-4 fw-bold">
+    <div id="date"></div>
+    <div id="clock"></div>
+  </div>
+
+<div id="container">
+  <h1 class="text-center mt-5">Books Library</h1>
+  <div id="books"></div>
+</div>
+
+<form class="addbook d-flex justify-content-center">
+  <h2 class="mt-5 text-center">Add a New Book</h2>
+  <div class="d-flex justify-content-center mt-5">
+    <label for="titulo"></label>
+    <input class="w-50 p-1" type="text" id="titulo" placeholder="Title of Book">
+  </div>
+  <div class="mt-3 d-flex justify-content-center">
+    <label for="autor"></label>
+    <input  class="w-50 p-1" type="text" id="autor" placeholder="Author of Book">
+  </div>
+  <div class="d-flex justify-content-center">
+  <button id="btn">Add</button>
+</div>
+</form>
```

Comment:
- [ ] Please make your form match the given [wireframe](https://github.com/microverseinc/curriculum-javascript/blob/main/books/images/awesome_books_full_website.png) now it looks like this :point_down: 

![image](https://user-images.githubusercontent.com/75126481/165755748-d9587bf2-c293-4391-848b-9bbe14988c4c.png)

- Original design :point_down: 

![image](https://user-images.githubusercontent.com/75126481/165755839-36d27c91-7717-4884-bdad-ebb2da86adef.png)